app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def update_comment_like
+    post.increment!(likes_counter)
+  end
+end
```

Comment:
It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save `to call the #update_comment_like method every time a new post has been saved to database, then make the update_comment_like method private 👍🏽